TMessagesProj/build.gradle

```diff
@@ -28,13 +27,14 @@ configurations {
 }
 
 dependencies {
-    compile 'com.android.support:support-core-ui:25.3.0'
-    compile 'com.android.support:support-compat:25.3.0'
-    compile 'com.android.support:support-core-utils:25.3.0'
-    compile 'com.android.support:support-v13:25.3.0'
-    compile 'com.android.support:palette-v7:25.3.0'
+    compile 'com.android.support:support-core-ui:25.3.1'
+    compile 'com.android.support:support-compat:25.3.1'
+    compile 'com.android.support:support-core-utils:25.3.1'
+    compile 'com.android.support:support-v13:25.3.1'
+    compile 'com.android.support:palette-v7:25.3.1'
     compile 'com.googlecode.mp4parser:isoparser:1.0.6'
     compile 'com.stripe:stripe-android:2.0.2'
+    compile fileTree(dir: "$buildDir/native-libs", include: 'native-libs.jar')
```

Comment:
that's a really interesting question, because this line, as well as the task 30 lines up... it was ditched by upstream in v1.4:
https://github.com/DrKLO/Telegram/commit/4ede31105599d0af566f4143c85b35767f98b59f#diff-3700629783fb7209df0b9e49538cd538

but Telegram-FOSS is carrying it still, and I'm not a native code guru. I think it would be safe to remove it and get closer to upstream, but that's a separate cleanup task